outputs.tf

```diff
@@ -1,3 +1,11 @@
 output "lambda" {
   value = aws_lambda_function.this
 }
+
+output "arn" {
+  value = aws_lambda_function.this.arn
+}
+
+output "function_name" {
+  value = aws_lambda_function.this.function_name
+}
```

Comment:
❓ Do we need these? They're covered by the first output. In the tests, we should be able to do something like this (assuming that it returns a map):
```
        lambda := terraform.Output(t, terraformOptions, "lambda")
	assert.Equal(t, "arn:aws:lambda:us-east-1:000000000000:function:use1-development-testLambda", lambda["arn"])
```

Reply:
good call, just needed to change `terraform.Output` to `terraform.OutputMap` and this works well 👍